pkg/handler/enqueue_mapped.go

```diff
@@ -68,6 +69,16 @@ func (e *EnqueueRequestsFromMapFunc) mapAndEnqueue(q workqueue.RateLimitingInter
 	}
 }
 
+// EnqueueRequestsFromMapFunc can inject fields into the mapper.
+
+// InjectFunc implements inject.Injector.
```

Comment:
I suggest you add something like below to ensure it implements the interface
```
var _ inject.Injector = EnqueueRequestsFromMapFunc(nil)
```

Reply:
+1